janmatyoo/ev-employee-onboarding-exercise
# PR for EVOBE-129

### Ticket
<https://phunts.atlassian.net/browse/EVOBE-129>


### Changes
- Added Unit Testing using Mocha and Chai

### Steps to Test
1. Pull branch
2. Go to the folder `backend`
4. Run the cli command `npm install` to install all the dependencies
5. Run the cli command `npm run test` to run the test cases
6. You should see five passing test cases


### Resources
- N/A
